R/junction-read-count.R

```diff
@@ -99,18 +135,33 @@ calculatePromoterReadCounts <- function(promoterAnnotationData, junctionFilePath
   if (any(!checkFile)) {
     stop(paste0('Error: Please specify valid junction file paths. The following file does not exist: ', junctionFilePaths[!checkFile]))
   }
+  
+  if (is.null(junctionFileLabels)) {
```

Comment:
maybe you can use the basenames, and if they are identical make them unique
tools::file_path_sans_ext(basename(junctionFilePaths))